### PR TITLE
msi_z690/hardware-matrix.md: replache ADL with RPL CPU in DDR5 unit

### PR DESCRIPTION
**Pull request by johanes2115**

### CODE REVIEW

**Comment by macpijan** [comment]
@johanes2115 Please fix commit message as explained in 1on1 discussion.

Also, please fix your git config:

`Jan baziuj <jan.baziuk@3mdeb.com>`

**Review by macpijan — CHANGES_REQUESTED** [review]
@johanes2115 Please fix commit message as explained in 1on1 discussion.

Also, please fix your git config:

```
Jan baziuj <jan.baziuk@3mdeb.com>
```

File: docs/variants/msi_z690/hardware-matrix.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ please check the [FAQ section](./faq.md).
 
 | Component                      | Description                                 |
 |--------------------------------|---------------------------------------------|
-| **CPU**                        | Intel Core i5-12600K 3.7                    |
+| **CPU**                        | Intel Core i5-13600K 3.5                    |
```
Comment (miczyg1) [inline docs/variants/msi_z690/hardware-matrix.md:60]: Again what is this 3.5?

Why Noctua is in the CPU section? It seems like some copy-pasta error in the whole file

**Comment by miczyg1** [comment]
@johanes2115 any chance you will address our comments?

File: docs/variants/msi_z690/hardware-matrix.md (hunk 1 above)
Comment (johanes2115) [reply docs/variants/msi_z690/hardware-matrix.md]: It was like that before my changes, but I fixed it and some other minor things in c8b8d568e72ec09090576989d4b76fca0f49a370

**Review by miczyg1 — APPROVED** [review]

**Comment by miczyg1** [comment]
> @johanes2115 Please fix commit message as explained in 1on1 discussion.
> 
> Also, please fix your git config:
> 
> ```
> Jan baziuj <jan.baziuk@3mdeb.com>
> ```

I see it has been fixed. MErging

**Pull request merged**